docs/source/inference.mdx

```diff
@@ -0,0 +1,109 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Optimum Inference with OpenVINO
+
+Optimum Intel can be used to load optimized models from the [Hugging Face Hub](hf.co/models) and create pipelines to run inference with OpenVINO Runtime without rewriting your APIs.
+
+## Switching from Transformers to Optimum Inference
+
+The Optimum Inference models are API compatible with Hugging Face Transformers models. This means you can just replace your `AutoModelForXxx` class with the corresponding `OVModelForXxx` class.
+To perform inference from a vanilla Transformers models, which will be later converted to an OpenVINO IR, set `from_transformers=True` when loading the model with the `from_pretrained()` method. 
+
+Here is an example on how to perform inference with OpenVINO Runtime for a text classification class:
+
+```diff
+-from transformers import AutoModelForSequenceClassification
++from optimum.intel.openvino import OVModelForSequenceClassification
+from transformers import AutoTokenizer, pipeline
+model_id = "distilbert-base-uncased-finetuned-sst-2-english"
+-model = AutoModelForSequenceClassification.from_pretrained(model_id)
++model = OVModelForSequenceClassification.from_pretrained(model_id, from_transformers=True)
+tokenizer = AutoTokenizer.from_pretrained(model_id)
+cls_pipe = pipeline("text-classification", model=model, tokenizer=tokenizer)
+result = cls_pipe("He's a dreadful magician.")
+[{'label': 'NEGATIVE', 'score': 0.9919503927230835}]
+```
+
+To easily save the resulting model, you can use the `save_pretrained()` method, which will save both the BIN and XML files describing the graph.
+
+```python
+# Save the converted model
+save_directory = "a_local_path"
+model.save_pretrained(save_directory)
+tokenizer.save_pretrained(save_directory)
+```
+
+By default, `OVModelForXxx` support dynamic shapes, enabling inputs of every shapes. To decrease latency, static shapes can be enabled by giving the desired inputs shapes.
+
+```python
+# Fix the batch size to 1 and the sequence length to 9
+model.reshape(1, 9)
+```
+
+Currently, OpenVINO only supports static shapes for inference on GPU. FP16 precision can also be enabled in order to further decrease latency.
+
+```python
+# Fix the batch size to 1 and the sequence length to 9
+model.reshape(1, 9)
```

Comment:
It makes sense in general for GPU. But in order to make this more valuable for the user, we need to show how to use the padding options of the tokenizer in order to infer with static shapes.